legate/driver/command.py

```diff
@@ -360,27 +367,23 @@ def cmd_eager_alloc(
     return ("-lg:eager_alloc_percentage", str(eager_alloc))
 
 
+def cmd_user_script(
+    config: ConfigProtocol, system: System, launcher: Launcher
+) -> CommandPart:
+    return () if config.user_script is None else (config.user_script,)
+
+
 def cmd_user_opts(
     config: ConfigProtocol, system: System, launcher: Launcher
 ) -> CommandPart:
     return config.user_opts
 
 
-CMD_PARTS = (
-    cmd_bind,
-    cmd_rlwrap,
-    cmd_gdb,
-    cmd_cuda_gdb,
-    cmd_nvprof,
-    cmd_nsys,
-    # Add memcheck right before the binary
-    cmd_memcheck,
-    # Now we're ready to build the actual command to run
-    cmd_legion,
+_CMD_PARTS_SHARED = (
     # This has to go before script name
     cmd_nocr,
     cmd_module,
```

Comment:
I think these two options should still be legion_python-only. `--nocr` is consumed only by `legion_python_main`, not `legion_canonical_python_main`. `cmd_module` sets `-m`, and it's too late to consume that at this point.

Reply:
Actually, we do have `--nocr` in canonical python, https://gitlab.com/StanfordLegion/legion/-/blob/master/bindings/python/canonical_python.cc#L35-39

Reply:
OK, IIUC `cmd_module` is still not suitable for the canonical case (and correspondingly the `--module` and `--cprofile` flags are `legion_python`-only).

Reply:
Yes if someone is using standard python, those would be `-m <module>` and `-m cProfile <module>` respectively

Reply:
Yes, you are right. I have removed `--module` and `--cprofile`. I have also enabled `--verbose`  and  `--dry-run` for canonical python, please check this commit https://github.com/nv-legate/legate.core/pull/539/commits/92d8f3fb14b6c36c6da0c67c5561e00af25a843e